test/features/login_redirect.feature

```diff
@@ -2,19 +2,15 @@
 Feature: Login Redirection
 
     @dashboard_login
-    Scenario: As an unauthenticated user, when I visit the /dashboard URL I see the login page
+    Scenario: As an unauthenticated user, when I visit the dashboard URL I see the login page
         Given "Unauthenticated" as the persona
         When I visit "/dashboard"
         Then I should see an element with xpath "//h1[contains(string(), 'Login')]"
-
-    @dashboard_login
-    Scenario: As an unauthenticated user, when I visit the /dashboard/ URL I see the login page
```

Comment:
given when then,
should not have another when then, re-introduce the scenario again.

Reply:
At a high level, though, it's the same scenario. The only difference is the trailing slash.

Could make it a parameterised scenario, I suppose?